### DISCUSSION TITLE
hardware env

### DISCUSSION THREAD

**Discussion by wux024** [talmolab/sleap#1648] [Help!]
What are the hardware requirements for reasoning at 600 FPS?

**Comment by Marielamend** [comment]
Hello @wux024!

There was a similar inquiry to yours that can give you some more insight (#959):

>**GPUs**:
We've done most of our testing on an Titan RTX, but that's way overkill, especially for realtime. Anything with >8 GB of memory should be enough, especially for inference. Any of the 30xx series should have enough CUDA cores to get your realtime latency pretty low.

Here is a [section of our paper](https://www.nature.com/articles/s41592-022-01426-1#:~:text=The%20acquisition%20computers%20were%20custom%2Dbuilt%20workstations%20with%20Intel%20i7%2D8700K%20central%20processing%20units%20(CPUs)%2C%2064%E2%80%89GB%20of%20RAM%2C%20a%20Samsung%20860%20Evo%20Series%204%2DTB%20SSD%20for%20data%20and%20an%20EVGA%20GeForce%20GTX%201080%20Ti%2011%2DGB%20GPU.) that describes the hardware that was utilized to run inference on up to 804 frames per second. 

Hope this is helpful but if you have any other questions let us know!

Best,
Mariela
